discovery-infra/tests/conftest.py

```diff
@@ -100,6 +115,17 @@ def get_api_client(offline_token=env_variables['offline_token'], **kwargs):
     return assisted_service_api.create_client(url, offline_token, **kwargs)
 
 
+def get_available_openshift_versions() -> List[str]:
+    available_versions = list(get_api_client().get_openshift_versions().keys())
+    specific_version = env_variables["openshift_version"]
+    if specific_version:
```

Comment:
I think this one is always true

Reply:
utils.get_openshift_version(default=None) returns None as default value